MCU_Microchip_PIC16.lib

```diff
@@ -1,6 +1,23 @@
 EESchema-LIBRARY Version 2.4
 #encoding utf-8
 #
+# ExamplePN
```

Comment:
You have added a ExamplePN device. I guess this was accidental. You should delete it